### PR TITLE
vendor pydantic

### PR DESCRIPTION
**Pull request by dbarrosop**
Relates to #480

### CODE REVIEW

**Comment by dbarrosop** [comment]
Done, not sure I like it though. Not sure what the purpose of the leading `_` is and, to make matters worse, not the vendor folder is shown first in the PR

**Comment by dmfigol** [comment]
> Done, not sure I like it though. Not sure what the purpose of the leading `_` is and, to make matters worse, not the vendor folder is shown first in the PR

to signal to users that they shouldn't import it / rely on it

File: pyproject.toml [hunk 1]
```diff
@@ -39,6 +38,9 @@ sphinx-issues = { version = "^1.2", optional = true }
 typing_extensions = "^3.7"
 ncclient = "^0.6.6"
 
+# for pydantic
+dataclasses = {version = "^0.7", python = "3.6"}
```
Comment (dmfigol) [inline pyproject.toml]: I think it should be `python = "~3.6"` according to
https://python-poetry.org/docs/dependency-specification/#python-restricted-dependencies

Comment (dbarrosop) [reply pyproject.toml]: fixed

**Pull request merged**